### PR TITLE
📝 Write a skeleton Q&A blog post for introducing Logan as chair

### PR DESCRIPTION
**Pull request by drewbrew**
As a way of introducing Logan to the community as conference chair, I wrote a brief post including his DEFNA bio and a link to his Twitter account with a few questions that I think would help.  Feel free to add more (and a proper header image).

### CODE REVIEW

File: _posts/2022-06-02-introducing-conference-chair-logan.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+category: General
+date: 2022-06-02 12:00:00
+layout: post
+title: "Introducing Logan Kilpatrick, the 2022 DjangoCon US Chair"
+---
+
+Announcing our chair for DjangoCon US 2022, [Logan Kilpatrick](https://twitter.com/OfficialLoganK)!
+
+From his [DEFNA bio](https://www.defna.org/about/):
+> Logan Kilpatrick is a Senior Technology Advocate at PathAI where he leads Machine Learning and Open Source Advocacy as well as the Developer Community Advocate for the Julia Programming Language. He has also been using and teaching Django at Harvard Universities Division of Continuing Education since early 2020. Outside of work and teaching, Logan is on the Board of Directors at NumFOCUS, the organization behind open source projects like Jupyter, NumPy, Pandas, and more. Open Source and communities are near and dear to Logan so you might run into him giving a talk, tinkering on some new project, or trying to recruit new contributors across the open source ecosystem.
```
Comment (drewbrew) [inline _posts/2022-06-02-introducing-conference-chair-logan.md]: Also I have no idea how the blockquote will render, so please adjust as needed

**Comment by logankilpatrick** [comment]
@drewbrew Let me know if you think this looks okay

File: _posts/2022-06-02-introducing-conference-chair-logan.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+> LK: "The biggest difference I have seen from traditional academic classes and more professional learning tracks is that in many academic settings, there is much more support. One of the biggest issue with Massive Open Online Courses (MOOC’s) and other "professional” learning tracks is that you are more or less on your own to figure it out, which in some sense is a great skill to practice, but I do find the traditional academic classroom to be an enabling force for those who might not be comfortable to just jump in and learn something like Django. I know I probably wasn't ready to do something like that had I not been in the confines of an academic classroom."
+
+## What sort of hobbies do you enjoy when you're away from the computer?
+
+> LK: "In the ever fleeting hours away from my computer, I try to be as active as I can be. I recently took up golf and have really enjoyed the sport for the ability to get out into nature and be totally disconnected from technology. I have also gotten into formula 1 over the last 2 years and I have really enjoyed watching the races and such."
```
Comment (drewbrew) [inline _posts/2022-06-02-introducing-conference-chair-logan.md]:
```suggestion
> LK: "In the ever fleeting hours away from my computer, I try to be as active as I can be. I recently took up golf and have really enjoyed the sport for the ability to get out into nature and be totally disconnected from technology. I have also gotten into Formula 1 over the last 2 years and I have really enjoyed watching the races and such."
```

File: _posts/2022-06-02-introducing-conference-chair-logan.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+author: DjangoCon US Organizers
+category: General
+date: 2022-06-06 12:00:00
```
Comment (drewbrew) [inline _posts/2022-06-02-introducing-conference-chair-logan.md]:
```suggestion
date: 2022-06-04 12:00:00
```

**Comment by drewbrew** [comment]
Looks great. I'll check on my windows box today to make sure it renders
well and then merge it. Thanks!

**Pull request merged**